packages/program/src/contract-call-script.ts

```diff
@@ -34,17 +31,14 @@ export const contractCallScript = new ScriptRequest<ContractCall[], Uint8Array[]
   // Script to call the contract
   contractCallScriptBin,
   (contractCalls) => {
-    const scriptDataCoder = new AbiCoder().getCoder(SCRIPT_INPUTS) as StructCoder<any>;
-    const callSlotsLength = (scriptDataCoder.coders.calls as ArrayCoder<any>).length;
-
-    if (contractCalls.length > callSlotsLength) {
-      throw new Error(`At most ${callSlotsLength} calls are supported`);
+    if (contractCalls.length > 5) {
+      throw new Error(`At most ${5} calls are supported`);
     }
```

Comment:
Looks like we were reading the number of max calls allowed from the script data coder earlier, but we are not anymore. Why is this the case?

Reply:
It was an "I'll deal with it later" but never did thing. I'll fix it now.